docs/source/iroha_2_whitepaper.md

```diff
@@ -1,137 +1,105 @@
+* Voting peers (participate in consensus)
+* Normal peer (receives and relays blocks, but does not participate in consensus;
+however, normal peers do validate all received data, with the mantra of *don't trust, verify*)
 
-### 2.2. Membership service
+#### 2.1.2. Peers discovery and membership
 
-Membership is provided in a decentralized way, on ledger. By default 2*f*+1 signatures are needed to confirm adding or removing nodes to or from the network. If nodes drop out and are unresponsive for "too long," then peers will automatically execute a remove peer smart contract to remove the peer from the consensus process.
+Membership is provided in a decentralized way, on ledger.
+By default 2*f*+1 signatures are needed to confirm adding or removing peers to or from the network.
+If peers drop out and are unresponsive for "too long,"
+then other peers will automatically execute a remove peer smart contract to remove the peer from the consensus process.
 
-### 2.3. Cryptography
+### 2.2. Cryptography
 
 We use [Hyperledger Ursa](https://github.com/hyperledger/ursa).
 
-### 2.4. Data Model
+### 2.3. Data Model
 
-Iroha uses a simple data model made up of domains, peers, accounts, assets, signatories, and permissions, as shown in the figure below:
+Iroha uses a simple data model made up of domains, peers, accounts, assets and signatories,
+as shown in the figure below:
 
 ```
                 has one
-   +--------------------------------+
-   |                                |
-   |     +------------+             |
-   |     |Domain      |             |
-   |     +---------+  |             |
-+--+-+   ||Asset(s)|  |             |
-|Peer|   +---------+  |             |
-+--+-+   |            |             |
-   |     +------------+             |
-   |     ||Account(s)||  has  +-----v-----+
-   |     |-------------------->Signatories|
-   |     +------------+       +-----------+
-   |                |
-   |                |    has  +-------------+
-   |                +--------->Permission(s)|
-   |                          +-----^-------+
-   |                                |
-   |                     has        |
-   +--------------------------------+
+   +-------------------------------------+
+   |                                     |
+   |     +-----------------+             |
+   |     |Domain           |             |
+   |     +--------------+  |             |
+   |     ||Asset        |  |             |
++--+-+   ||Definition(s)|  |             |
+|Peer|   +--------------+  |             |
++--+-+   |                 |             |
+   |     +------------+    |             |
+   |     ||Account(s)||    | has   +-----v-----+
+   |     |------------------------->Signatories|
+   |     +-----------------+       +-----------+
+   |                       |             |
+   |                       |  has  +--------+
+   |                       +------->Asset(s)|
```

Comment:
Shouldn't the concept of permissions be here somewhere?

Reply:
In current impl we have permissions in form of assets, this solution is not accepted and we will discuss it in https://wiki.hyperledger.org/display/iroha/Permissions 

I decided to update current model in the whitepaper to be aligned with current state